sencha-workspace/packages/slate-cbl/src/view/student/skill/OverviewWindow.js

```diff
@@ -0,0 +1,87 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.student.skill.OverviewWindow', {
+    extend: 'Ext.window.Window',
+    xtype: 'slate-cbl-student-skill-overviewwindow',
+    requires: [
+        'Slate.cbl.view.student.skill.OverviewWindowController',
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    controller: 'slate-cbl-student-skill-overviewwindow',
+
+    config: {
+        student: null,
+        competency: null,
+        skill: null
+    },
+
+    title: 'Standard Overview',
+    width: 700,
+    minWidth: 700,
+    shadow: 'frame',
+    center: true,
+    fixed: true,
```

Comment:
This is the fixed version. Behavior is a bit more sensible than the teacher one.